sql/core/src/main/java/org/apache/spark/sql/execution/streaming/state/StateMessage.java

```diff
@@ -217,12 +217,12 @@ public interface StateRequestOrBuilder extends
    * Protobuf type {@code org.apache.spark.sql.execution.streaming.state.StateRequest}
    */
   public static final class StateRequest extends
-      com.google.protobuf.GeneratedMessageV3 implements
+      com.google.protobuf.GeneratedMessage implements
```

Comment:
I submitted a PR to change to use a `maven/sbt` plugin to generation of this file: https://github.com/apache/spark/pull/48654